ext/ffi_c/Function.c

```diff
@@ -890,6 +900,7 @@ rbffi_Function_Init(VALUE moduleFFI)
     rb_define_method(rbffi_FunctionClass, "call", function_call, -1);
     rb_define_method(rbffi_FunctionClass, "attach", function_attach, 2);
     rb_define_method(rbffi_FunctionClass, "free", function_release, 0);
+    rb_define_method(rbffi_FunctionClass, "type", function_type, 0);
```

Comment:
`FFI::FunctionInfo` is an internal class only. It is used for internal optimization, but shouldn't be exposed anywhere. I think `FFI::Function` should get `return_type` and `param_types` instead.

Reply:
With the latest changes (see below) this isn't necessary anymore, I've removed it.